src/editor_serie/EditorSerieController.cpp

```diff
@@ -147,6 +149,15 @@ EditorSerieController::EditorSerieController() : Controller()
         ERROR_NEW_SERIE_POSITION_Y
     );
 
+    levelErrorLabel.setFont(serieNameLabelFont);
+    levelErrorLabel.setCharacterSize(constants::Fonts::SIZE_MESSAGE_FONT);
+    levelErrorLabel.setColor(errorLabelColor);
+    levelErrorLabel.setString(STRING_NEW_LEVEL_ERROR);
+    levelErrorLabel.setPosition(
+        ERROR_NEW_SERIE_POSITION_X,
+        ERROR_NEW_SERIE_POSITION_Y
```

Comment:
this position is the same for the new serie error and new level error, update the name